scripts/vote_sr_v2.py

```diff
@@ -0,0 +1,421 @@
+"""
```

Comment:
Lets rename to upgrage_sr_v2.py instead of vote_.... to include snapshot tests over all changes

Reply:
good point, renamed.